recipes/pyspelling/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pyspelling" %}
+{% set version = "2.7.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyspelling-{{ version }}.tar.gz
+  sha256: ba1d737d66c3f8992a183c5c7515bc9cb2ee5d2103310a969af32331bb70ec2a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyspelling=pyspelling.__main__:main
+    - pyspelling3.9=pyspelling.__main__:main
```

Comment:
Does it always provide a 3.9 suffixed entrypoint or is this dependent on the Python version?

Reply:
I need to check the code. I'm pretty sure it is python dependant and, in that case, we'll need to patch it to make it truly noarch :-(

Reply:
Oh fun! https://github.com/facelessuser/pyspelling/blob/3edf5857c972970a607df77ad5cc14fc01ab3949/setup.py#L48

Reply:
@xhochy we remove versioned entrypoints in other places too. While I don't like to maintain patches, I believe this is the best I can do for now. What do you think?

Reply:
Sounds fine.